src/app/client/src/app/modules/dashboard/components/data-chart/data-chart.component.html

```diff
@@ -18,7 +18,7 @@
         </div>
       </div>
 
-      <div class="mr-8" *ngIf="chartType === 'bar' || chartType === 'line'" [ngClass]="{'hide': hideElements}">
+      <div class="mr-8" *ngIf="chartType === 'bar' || chartType === 'line' && !bigNumberCharts.length" [ngClass]="{'hide': hideElements}">
```

Comment:
condition should be restricted to only bar and line. Reason being  event line and bar charts can have bigNumber at the top